app/models/chargeback_rate.rb

```diff
@@ -109,11 +109,11 @@ def self.seed
         else
           if fix_mtime > rec.created_on
             _log.info("Updating [#{cbr[:description]}] with guid=[#{cbr[:guid]}]")
-            rec.update_attributes(cbr)
+            rec.update(cbr)
             rec.chargeback_rate_details.clear
             rec.chargeback_rate_details.create(rates)
             rec.created_on = fix_mtime
-            rec.save
+            rec.save!
```

Comment:
this was the only real change in this PR

(I ignored the changes in the specs)